example-package-dealer.py

```diff
@@ -10,7 +10,7 @@
 # RUN: git clone https://github.com/apple/example-package-dealer %t.dir/dealer
 
 # RUN: rm -rf %t.dir/dealer/.build
-# RUN: %{swift} build --package-path %t.dir/dealer 2>&1 | tee %t.build-log
+# RUN: %{swift-build} --package-path %t.dir/dealer 2>&1 | tee %t.build-log
```

Comment:
I don't think we should do this because then we're not testing swift's subtool invocation.

Reply:
We're testing the swift's subtool invocation by default, when not passing `swiftpm_build`.

Reply:
Ah okay. This repo doesn't have PR testing so maybe create a dummy PR on swift and then do a cross-repo testing?